### PR TITLE
tutorials:alerting:get-started-p2

### PR DESCRIPTION
**Pull request by tonypowa**
This is part 2 of the Get started with Grafana Alerting tutorials

Objectives:

- Configure an alert rule that returns more than one alert instance
- Create notification policies that route firing alert instances to different contact points
- Use labels to match alert instances and notification policies

### CODE REVIEW

**Comment by tonypowa** [comment]
testing locally

**Comment by tonypowa** [comment]
good to go @brendamuir ! 😉 

Shouldn't
I
Enjoy
Some
Time
Asleep?

File: docs/sources/tutorials/alerting-get-started-pt2/index.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+- Configure an alert rule that returns more than one alert instance
+- Create notification policies that route firing alert instances to different contact points
+- Use labels to match alert instances and notification policies
+
+Learning about alert instances and notification policies is useful if you have more than one contact point in your organization, or if your alert rule returns a number of metrics that you want to handle separately by routing each alert instance to a specific contact point. The tutorial will introduce each concept, followed by how to apply both concepts in a real-world scenario.
+
+## Alert instances
+
+An [alert instance](https://grafana.com/docs/grafana/latest/alerting/fundamentals/#alert-instances) is an event that matches a metric returned by a query of an alert rule.
```
Comment (brendamuir) [inline docs/sources/tutorials/alerting-get-started-pt2/index.md:36]:
```suggestion
An [alert instance](https://grafana.com/docs/grafana/latest/alerting/fundamentals/#alert-instances) is an event that matches a metric returned by an alert rule query.
```

File: docs/sources/tutorials/alerting-get-started-pt2/index.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+Let's consider a scenario where you're monitoring website traffic using Grafana. You've set up an alert rule to trigger an alert instance if the number of page views exceeds a certain threshold (more than `1000` page views) within a specific time period, say, over the past `5` minutes.
+
+If the query returns more than one time-series, each time-series represents a different metric or aspect being monitored. In this case, the alert rule is applied individually to each time-series.
+
+{{< figure src="/media/docs/alerting/get-started-digram-instance-grey.png" max-width="1200px" caption="Alert Instances in the Context of an Alert Rule" >}}
+
+In this scenario, each time-series is evaluated independently against the alert rule. It results in the creation of an alert instance for each time-series. The time-series corresponding to the desktop page views meets the threshold and, therefore, results in an alert instance in **Firing** state for which an alert notification will be sent. The mobile alert instance state remains **Normal**.
```
Comment (brendamuir) [inline docs/sources/tutorials/alerting-get-started-pt2/index.md:44]:
```suggestion
In this scenario, each time-series is evaluated independently against the alert rule. It results in the creation of an alert instance for each time-series. The time-series corresponding to the desktop page views meets the threshold and, therefore, results in an alert instance in **Firing** state for which an alert notification is sent. The mobile alert instance state remains **Normal**.
```

File: docs/sources/tutorials/alerting-get-started-pt2/index.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+## Notification policies
+
+[Notification policies](https://grafana.com/docs/grafana/latest/alerting/fundamentals/notifications/notification-policies/) route alerts to different communication channels, reducing alert noise and providing control over when and how alerts are sent. For example, you might use notification policies to ensure that critical alerts about server downtime are sent immediately to the on-call engineer. Another use case could be routing performance alerts to the development team for review and action.
+
+Key Characteristics:
+
+- Route alert notifications by matching alerts and policies with labels
+- Manage when to send notifications
+
+{{< figure src="/media/docs/alerting/get-started-notification-policy-tree-combo.png" max-width="1200px" caption="Routing of alerts with notification policies" >}}
```
Comment (brendamuir) [inline docs/sources/tutorials/alerting-get-started-pt2/index.md:55]: re accessibility - every image should have alt text: {{< figure alt="<ALT TEXT>"  src="/media/<PATH TO IMAGE>" >}}

File: docs/sources/tutorials/alerting-get-started-pt2/index.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+1. Remove the ‘B’ **Reduce expression** (click the bin icon). The Reduce expression is default, and in this case, is not required since the queried data is already reduced. Note that the Threshold expression is now your **Alert condition**.
+1. In the ‘C’ **Threshold expression**:
+    - Change the **Input** to ‘**A**’ to select the data source.
+    - Enter `1000` as the threshold value. This is the value above which the alert rule should trigger.
+1. Click **Preview** to run the queries.
+
+It should return two series.`desktop` in Firing state, and `mobile` in Normal state. The values `1`, and `0` mean that the condition is either `true` or `false`.
+
+{{< figure src="/media/docs/alerting/get-started-expression-instances.png" max-width="1200px" caption="Preview of a query returning two alert instances in Grafana." >}}
```
Comment (brendamuir) [inline docs/sources/tutorials/alerting-get-started-pt2/index.md:124]: same as above

File: docs/sources/tutorials/alerting-get-started-pt2/index.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+### Configure labels and notifications
+
+In this section, you can select how you want to route your alert instances. Since we want to route by notification policy, we need to ensure that the labels match the alert instance.
+
+1. Choose **Use notification policy**.
+1. Click **Preview routing**. Based on the existing labels, you should see a preview of what policies are matching with the alerts. There should be two alert instances matching the labels that were previously setup in each notification policy: `device=desktop`, `device=mobile`.
+
+    These [types of labels](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/annotation-label/#label-types) are generated by the data source query and they can be leveraged to match our notification policies without needing to manually add them to the alert rule.
+
+    {{< figure src="/media/docs/alerting/get-started-alert-instace-routing-prev.png" max-width="1200px" caption="Routing preview of matched notification policies" >}}
```
Comment (brendamuir) [inline docs/sources/tutorials/alerting-get-started-pt2/index.md:146]: same as above

**Review by brendamuir — APPROVED** [review]

**Review by brendamuir — APPROVED** [review]

**Pull request merged**